python/lsst/pipe/tasks/postprocess.py

```diff
@@ -1417,6 +1417,10 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
         inputRefs = reorderRefs(inputRefs, detectorOrder, dataIdKey="detector")
         self.log.warning(f"{visits} inputRefs: {inputRefs}")
         inputs = butlerQC.get(inputRefs)
+
+        rows = [handle.get(component="rowcount") for handle in inputs["inputCatalogs"]]
+        self.log.warning(f"rows: {rows}")
```

Comment:
```suggestion
        self.log.warning("rows: %d", rows)
```

Reply:
Thanks, but these logs are all just temporary to diagnose the issue that we're discussing in the thread. Having warnings here is definitely bad. :)